### ISSUE TITLE
Feature Request: Remind me button in RocketChat-mobile-app

### ISSUE THREAD

**Issue by saschafoerster** [AddReminders/Reminders-in-RocketChat#5]
I am missing the remind-me-button in the context menu of messages in the mobile app of Rocket.Chat. Most of the times I am "on the go" when I want to be reminded later, it seems to me the use case I would appreciate most. 
I guess there is some Limitation in Rocket.Chat-app engine. But hopefully this can be changed as well. Thank you for creating this helpful tool!

**Comment by saschafoerster** [comment]
![image](https://user-images.githubusercontent.com/1153741/166101693-e76a9df2-8d08-4865-acbe-a0c478bffc4f.jpeg)

That's where I am missing it. 📷

**Comment by saschafoerster** [comment]
And of course. A workaround could be to type /remind into this chat and create another reminder at that place. But it's even better if the message that you want be remind of is linked.

**Comment by murtaza98** [comment]
> And of course. A workaround could be to type /remind into this chat and create another reminder at that place. But it's even better if the message that you want be remind of is linked.

Yeah, I agree, but unfortunately not much can be done from the app side for this issue. This is more of an issue on the Rocket.Chat's react native app, I'm afraid

PS: I'd already raised this issue earlier over here https://github.com/RocketChat/Rocket.Chat.ReactNative/issues/3723

**Comment by tuxmartin** [comment]
Support in mobile app will be great.

Most often you need to create reminder right on your mobile.
